### PR TITLE
Uplink fix

### PR DESCRIPTION
**Pull request by neersighted**

### CODE REVIEW

**Comment by KazeEspada** [comment]
That is a lot more changes than a simple uplink fix.

**Comment by neersighted** [comment]
Whoop it appears to have #14928 included. Let me rebase that.

File: code/modules/uplink/uplink_item.dm [hunk 1]
```diff
@@ -1063,10 +1065,11 @@ var/list/uplink_items = list() // Global list so we only initialize this once.
 	name = "Syndicate Surplus Crate"
 	desc = "A dusty crate from the back of the Syndicate warehouse. Rumored to contain a valuable assortion of items, \
 			but you never know. Contents are sorted to always be worth 50 TC."
+	item = /obj/structure/closet/crate
```
Comment (Razharas) [inline code/modules/uplink/uplink_item.dm]: What is this supposed to do

File: code/modules/uplink/uplink_item.dm [hunk 2]
```diff
@@ -1090,6 +1093,7 @@ var/list/uplink_items = list() // Global list so we only initialize this once.
 	name = "Random Item"
 	desc = "Picking this will purchase a random item. Useful if you have some TC to spare or if you haven't \
 			decided on a strategy yet."
+	item = /obj/item/weapon/paper
```
Comment (Razharas) [inline code/modules/uplink/uplink_item.dm]: And this?

File: code/modules/uplink/uplink_item.dm (hunk 1 above)
Comment (neersighted) [inline code/modules/uplink/uplink_item.dm]: It just needs a placeholder item in order to spawn, the actual `spawn_item` proc will ignore this and create the real item.

File: code/modules/uplink/uplink_item.dm (hunk 2 above)
Comment (neersighted) [inline code/modules/uplink/uplink_item.dm]: See above

**Pull request merged**